.github/workflows/greetings.yml

```diff
@@ -0,0 +1,19 @@
+name: Greetings
+
+on: [pull_request]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        pr-message: >
+          Thank you so much for your PR! We'll do our best to review your submission as soon as possible.
+
+          If you don't hear from us within 3 business days, please ping @thepracticaldev/oss.
```

Comment:
Will this ping you every time a PR is opened?

Reply:
No, it would be a first time only https://github.com/actions/first-interaction